### ISSUE TITLE
The method message_post need the subject and body be translatable

### ISSUE THREAD

**Issue by JesusZapata** [OCA/pylint-odoo#176]
The method `self.message_post` need the parameter `subject` and `body` be translatable

For example, the follow method raise the check `translation-required` because the parameter `subject` and `body` not is instance of `_` function

```python
self.message_post(
    subject='Error when trying to generate the invoice',  # translation-required
    body='<h2>%s</h2><hr>%s</hr>' % (  # translation-required
        _('Error when trying to generate the invoice'),
        _('The invoice should have a datetime set.')))
```

The correct way to this code is the follow

```python
self.message_post(
    subject=_('Error when trying to generate the invoice'),
    body=_('<h2>%s</h2><hr>%s</hr>' % (
        _('Error when trying to generate the invoice'),
        _('The invoice should have a datetime set.'))))
```
